modules/page_object_fxa_new_account.py

```diff
@@ -0,0 +1,37 @@
+from modules.page_base import BasePage
+
+
+class FxaNewAccount(BasePage):
+    """Page Object Model for FxA signup flow"""
+
+    URL_TEMPLATE = "{fxa_url}"
```

Comment:
Just wondering is this a placeholder? Or a regex? 😅

Reply:
This is a regex--because the url will have to be provided by a fixture, we have to just get the whole url here